### PR TITLE
README dead links - replaced

### PR DESCRIPTION
**Pull request by Manasi2001**
The dead links have been replaced. They are working fine now. Kindly review.

### CODE REVIEW

**Review by jbampton — COMMENTED** [review]
Check Markdown links is failing

**Comment by Manasi2001** [comment]
> Check Markdown links is failing

Ok, let me recheck.

**Comment by jbampton** [comment]
https://github.com/slurpcode/slurp/runs/6894546508?check_suite_focus=true

**Comment by Manasi2001** [comment]
Corrected that too @jbampton.

**Comment by jbampton** [comment]
https://github.com/tcort/markdown-link-check/search?q=403

**Comment by jbampton** [comment]
https://github.com/gaurav-nelson/github-action-markdown-link-check/search?q=403&type=issues

**Comment by Manasi2001** [comment]
Although the link is working just fine if I open it separately in a browser but somehow the 'Check Markdown links' is still failing. The link which I added wasn't posing this problem. Should I explore any other website and put its link @jbampton?

**Comment by jbampton** [comment]
https://github.com/openmrs/openmrs-esm-core/pull/412/files

A config file is needed

**Review by jbampton — CHANGES_REQUESTED** [review]
`mlc_config.json` should be in the root folder

File: .github/workflows/markdown-link-checker.yml [hunk 1]
```diff
@@ -13,3 +13,4 @@ jobs:
           check-modified-files-only: 'yes'
           use-quiet-mode: 'yes'
           use-verbose-mode: 'yes'
+          config-file: 'mlc_config.json'
```
Comment (jbampton) [inline .github/workflows/markdown-link-checker.yml:16]:
```suggestion
          check-modified-files-only: 'yes'
          config-file: 'mlc_config.json'
          use-quiet-mode: 'yes'
          use-verbose-mode: 'yes'
```

**Review by jbampton — APPROVED** [review]

**Pull request merged**